src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -92,67 +57,109 @@ export default {
 
       this.node.definition.targetRef = targetShape.component.node.definition;
       this.sourceShape.component.node.definition.get('outgoing').push(this.node.definition);
-      targetShape.component.node.definition.get('incoming').push(this.node.definition)
+      targetShape.component.node.definition.get('incoming').push(this.node.definition);
 
       this.updateWaypoints();
 
       targetShape.attr({
-        body: { fill: '#fff', cursor: 'move' },
+        body: { fill: `${this.defaultNodeColor}`, cursor: 'move' },
         label: { cursor: 'move' },
+
+        ".body": { fill: `${this.defaultNodeColor}`, cursor: 'move' },
+        ".label": { cursor: 'move' },
       });
 
       this.shape.listenTo(this.sourceShape, 'change:position', this.updateWaypoints);
       this.shape.listenTo(targetShape, 'change:position', this.updateWaypoints);
     },
+    checkExclusiveGateway() {
```

Comment:
When someone looks at this code, they may ask "Check what about exclusive gateway?" I think a better function name may be `isValidGatewayConnection`; this also hints that this function will return a boolean. Or you could add a comment here (which you might want to do anyway since gateway validation may seem confusing).